edgelet/edgelet-config/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2018"
 
 [dependencies]
 base64 = "0.9"
-config = "0.8"
+config = "0.9.2"
```

Comment:
What is this change required for?

Reply:
I believe I was running into some issues earlier with the config crate version 0.8 when I was using the Option<u32> change. Have reverted this now given how the new method seems to work with 0.8 too